test/system/irohad_test_data/config.sample

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Please get admin@test.priv, admin@test.pub, node0.priv, node0.pub, test@test.priv, test@test.pub back to example directory.

What was the reason to remove them?

Reply:
The reason is to separate the data needed for testing from the data deeded for examples. I found that admin keys in the example directory are also referenced in the iroha getting started guide, so I undo removing them. The rest files seem only needed for testing.